util/generator/src/templates/#report.org.template#

```diff
@@ -0,0 +1,35 @@
+#+author: {author}
+#+title:  {kind} report for {from} to {to}
+#+date:   {date}
+
+** Overview
+
+Thanks again to [[https://opensrcsec.com/][Open Source Security, inc]] and [[https://www.embecosm.com/][Embecosm]] for their ongoing support for this project.
+
+** Completed Activities
+{merged_prs}
+
+*** Contributors this week
+{contributors}
+
+*** Overall Task Status
+{task_status}
+
+*** Test Cases
+{test_cases}
+
+*** Bugs
+{bugs}
+
+*** Milestones Progress
+{milestones}
+
+*** Risks
+
+| Risk                           | Impact (1-3) | Likelihood (0-10) | Risk (I * L) | Mitigation                                     |
+|--------------------------------+--------------+-------------------+--------------+------------------------------------------------|
+| Missing GCC 13 upstream window |            2 |                 3 |            6 | Merge in GCC 14 and be proactive about reviews |
+
+** Planned Activities
+
+** Detailed changelog
```

Comment:
Weird naming on this file `util/generator/src/templates/#report.org.template#`. Looks like some emacs temp file. Am I right ?